docs/src/modules/components/Ad.js

```diff
@@ -98,7 +98,7 @@ function Ad(props) {
         if (
           document.querySelector('#carbonads a') &&
           document.querySelector('#carbonads a').getAttribute('href') ===
-            'https://material-ui-next.com/discover-more/backers'
+          'https://material-ui-next.com/discover-more/backers'
```

Comment:
Odd. That might be from a VSCode ESLint extension, but it's supposed to use the repo config...